pkg/codegen/templates/param-types.tmpl

```diff
@@ -1,6 +1,6 @@
 {{range .}}{{$opid := .OperationId}}
 {{range .TypeDefinitions}}
 // {{.TypeName}} defines parameters for {{$opid}}.
-type {{.TypeName}} {{.Schema.TypeDecl}}
+type {{.TypeName}} {{if and (opts.AliasTypes) (.CanAlias)}}={{end}} {{.Schema.TypeDecl}}
```

Comment:
if opts.AliasTypes=faalse or .CanAlias=false, you will generate an empty type alias, no?

`type TypeName`

Reply:
If one of them is false, the type will be 
`type TypeName .Schema.TypeDecl`
else it will be 
`type TypeName = .Schema.TypeDecl`